conda_vendor/conda_vendor.py

```diff
@@ -584,36 +607,36 @@ def vendor(
     """
     _green(f"Vendoring Local Channel for file: {file}", bold=False)
 
+    environment_file = Path(file)
+    environment_name = _get_environment_name(environment_file)
+
+    vendored_root = Path.cwd() / environment_name
```

Comment:
Is this method better than using os.path.join?

Reply:
yes.  i like it better.

Reply:
Does the "/" work for both linux and windows paths?

Reply:
yep.